app/views/scheduler/index.html.haml

```diff
@@ -9,8 +9,11 @@
           = text_field_tag :course_title, nil, placeholder: 'Add course instance', class: 'form-control'
 
         = form_for :semester, url: scheduler_path, method: :get, remote: true, html: {id: :semester_form} do |f|
-          = f.select :semester, %w(Fall Spring Summer).map { |str| [str, str.downcase] }
-          = f.select :year, options_for_select(2008..DateTime.now.year, DateTime.now.year)
+          = f.select :semester, options_for_select( |
+            %w(Fall Spring Summer).map {|str| [str, str.downcase]}, |
```

Comment:
Haml refuses to compile line breaks anywhere except for after commas, unless there is a pipe. Since there's no way to break this line under 80 chars with commas, I used the pipe, which denotes the continuation of the line.
Judging by Hound's response, the only way to fix this is with a helper method? Which sounds to me like it would make our code even less sane.